__docs__/wonder-blocks-icon-button/icon-button.stories.tsx

```diff
@@ -167,14 +168,57 @@ export const Variants: StoryComponentType = {
 };
 
 /**
- * An IconButton on a dark background.
- * Only the primary kind is allowed to have the `light` prop set to true.
+ * IconButton has a `color` that is either `default` (the default, as shown
+ * above) or `destructive` (as can seen below):
+ */
+export const WithColor: StoryComponentType = {
+    name: "Color",
+    render: () => (
+        <View style={styles.row}>
+            <IconButton
+                icon={minusCircle}
+                onClick={() => {}}
+                color="destructive"
+            />
+            <IconButton
+                icon={minusCircle}
+                onClick={() => {}}
+                kind="secondary"
+                color="destructive"
+            />
+            <IconButton
+                icon={minusCircle}
+                onClick={() => {}}
+                kind="tertiary"
+                color="destructive"
+            />
+            <IconButton
+                disabled={true}
+                icon={minusCircle}
+                aria-label="search"
+                onClick={(e) => console.log("Click!")}
+                color="destructive"
+            />
+        </View>
+    ),
+};
+
+/**
+ * An `IconButton` on a dark background. Only the primary kind is allowed to have
+ * the `light` prop set to true.
  */
 export const Light: StoryComponentType = {
     render: () => {
         return (
-            <View style={styles.dark}>
+            <View style={[styles.dark, styles.row]}>
+                <IconButton
+                    icon={magnifyingGlass}
+                    aria-label="search"
+                    light={true}
+                    onClick={(e) => console.log("Click!")}
+                />
                 <IconButton
+                    color="destructive"
```

Comment:
Is this supposed to look different than the default normally and on hover? It looks like the color only shows in the "active" state when clicking on the button.

Reply:
That's expected (the color only changes in the pressed state). You can see how that matches to what we do with `Button`: https://www.figma.com/file/VbVu3h2BpBhH80niq101MHHE/%F0%9F%92%A0-Main-Components?type=design&node-id=6396-5801&mode=design&t=Z5d5gJ1FZQ2VDI8L-4